src/content/docs/en/guides/backend/turso.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Turso & Astro
+description: Build locally with a SQLite file and deploy globally using Turso.
+type: backend
+service: Turso
+stub: false
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[Turso](https://turso.tech) is a distributed database built on libSQL, a fork of SQLite. It is optimized for low query latency, making it suitable for global applications.
+
+### Prerequisites
+
+- The [Turso CLI](https://docs.turso.tech/reference/turso-cli) installed and signed in
+- A [Turso](https://turso.tech) Database with schema
+- Your Database URL
+- An Access Token
+
+If you don't have an existing database and schema with Turso, create one and connect to the shell using the Turso CLI:
+
+```bash
+turso db create my-astro-db
+turso db shell my-astro-db
+```
+
+Then add the following:
+
+```sql
+CREATE TABLE posts (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    title TEXT,
+    slug TEXT UNIQUE,
+    content TEXT
+);
+```
```

Comment:
I think I would still suggest removing this, since you've said in the prerequisites that they SHOULD have this. So, this step to create one really isn't part of the point of connecting to Turso here.

Reply:
I'm happy to remove it but a pet peeve of my browsing "guides" in particular is that they always assume there's something you need to do beforehand to follow a guide. Totally happy to remove (and I will).

Reply:
That is literally what "Prerequisites" is for!  😄